### PR TITLE
Feature/325 create tutorial extract data from website save into excel and send an email

### PR DESCRIPTION
**Pull request by WiktoriaPrusik**

### CODE REVIEW

**Review by przegwar — APPROVED** [review]

**Review by przegwar — APPROVED** [review]

**Review by przegwar — APPROVED** [review]

File: Automation Examples/040.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+selenium.open type chrome url http://www.worldometers.info/world-population/population-by-country/
+window ✱Population✱ style maximize
+```
+
+#### 2. Create an excel file and fill it with headers
+
+The C# snippet `⊂♥headers.Count⊃` returns a number of `♥headers` elements. In each iteration of a loop, a header is appended to the excel.
+
+```G1ANT
+♥path = ‴C:\Users\dell\Desktop\Automation Anywhere\countries.xlsx‴
```
Comment (danieladamkaiser) [inline Automation Examples/040.md]: Dont use path from your pc

**Review by danieladamkaiser — APPROVED** [review]

File: Automation Examples/040.md (hunk 1 above)
Comment (WiktoriaPrusik) [reply Automation Examples/040.md]:
```suggestion
♥path = ‴♥environment⟦HOMEDRIVE⟧♥environment⟦HOMEPATH⟧\Documents\G1ANT.Robot\countries.xlsx‴
```

File: Automation Examples/040.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+Make sure to enable addon selenium from the View/Addons panel.
+
+```G1ANT
+selenium.open type chrome url http://www.worldometers.info/world-population/population-by-country/
+window ✱Population✱ style maximize
+```
+
+#### 2. Create an excel file and fill it with headers
+
+The C# snippet `⊂♥headers.Count⊃` returns a number of `♥headers` elements. In each iteration of a loop, a header is appended to the excel.
```
Comment (WiktoriaPrusik) [inline Automation Examples/040.md]:
```suggestion
The C# snippet `⊂♥headers.Count⊃` returns a number of `♥headers` elements. In each iteration of a loop, a header is appended to the excel. Using environment variables, the path will be correct on any computer.
```

File: Automation Examples/040.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+#### 4. Create a loop with no body but with the correct conditions
+
+`♥websiteRecordsNumber` must be of an integer structure because it will be compared to the `♥i` variable storing a number of a website record that is being currently processed. `♥excelRecordsNumber` is a number of records matchting the criteria from the "process description". So it is probably the best option here to use a while loop which should end if there are no more records on the website (`♥i` is equal to `♥websiteRecordsNumber`) or if `♥excelRecordsNumber` is 15.
+
+```G1ANT
+♥websiteRecordsNumber = ⟦integer⟧♥websiteRecordsNumber
+♥i = 1
+♥excelRecordsNumber = 1
+
+while ⊂♥i < ♥websiteRecordsNumber && ♥excelRecordsNumber < 16⊃
```
Comment (WiktoriaPrusik) [inline Automation Examples/040.md]:
```suggestion
while ⊂♥i < ♥websiteRecordsNumber && ♥excelRecordsNumber <= 15⊃
```

File: Automation Examples/040.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+#### 4. Create a loop with no body but with the correct conditions
+
+`♥websiteRecordsNumber` must be of an integer structure because it will be compared to the `♥i` variable storing a number of a website record that is being currently processed. `♥excelRecordsNumber` is a number of records matchting the criteria from the "process description". So it is probably the best option here to use a while loop which should end if there are no more records on the website (`♥i` is equal to `♥websiteRecordsNumber`) or if `♥excelRecordsNumber` is 15.
+
+```G1ANT
+♥websiteRecordsNumber = ⟦integer⟧♥websiteRecordsNumber
+♥i = 1
+♥excelRecordsNumber = 1
+
+while ⊂♥i < ♥websiteRecordsNumber && ♥excelRecordsNumber <= 15⊃
```
Comment (WiktoriaPrusik) [inline Automation Examples/040.md]:
```suggestion
while ⊂♥i != ♥websiteRecordsNumber && ♥excelRecordsNumber != 15⊃
```

**Pull request merged**